src/ray/raylet/worker_pool.cc

```diff
@@ -77,12 +84,12 @@ WorkerPool::~WorkerPool() {
   }
 }
 
-uint32_t WorkerPool::Size() const {
-  return static_cast<uint32_t>(actor_pool_.size() + pool_.size());
+uint32_t WorkerPool::Size(const Language &language) const {
```

Comment:
Maybe `GetSize` will get more in line with our cpp style?

I saw some get function names are prefix with `Get` some where, but some not.
How should we unify it?

Reply:
Google style prefers the no-get style: https://google.github.io/styleguide/cppguide.html#Function_Names, but it's not required.
> Accessors and mutators (get and set functions) may be named like variables. 

Most code in Ray follows this convention, which I think it good. But there's an exception in this PR. in `Worker` class, I named the getter function of `language_` as `GetLanguage`. The reason is because if I name it `Language`, it will override the definition of `enum class Language`.